From 422efcea69fe9a56c384e25074f135d6f5d78abe Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Wed, 10 Nov 2021 20:30:40 +0300
Subject: [PATCH] wine.inf: HACK: Spoof Win81 for msedgewebview2.exe.

A proper fix depends on other process window Vulkan rendering
and Direct Compositing implementation.

CW-Bug-ID: #19618

For FH5.
---
 loader/wine.inf.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 3c927c207a8..09a0b1b3eaf 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -4238,6 +4238,7 @@ HKCU,Software\Wine\AppDefaults\Paradox Launcher.exe\DllOverrides,"libglesv2",,"d
 HKCU,Software\Wine\AppDefaults\Nickelodeon All-Star Brawl.exe\DllOverrides,"winusb",,"disabled"
 HKCU,Software\Wine\AppDefaults\nw.exe\DllOverrides,"libglesv2",,"disabled"
 HKCU,Software\Wine\AppDefaults\gotg.exe\DllOverrides,"atiadlxx",,"builtin"
+HKCU,Software\Wine\AppDefaults\msedgewebview2.exe,"Version",,"win81"
 ;;Likely want *80 and *90 too, but those require removing Wine's manifest files.
 HKCU,Software\Wine\DllOverrides,"atl100",0x2,"native,builtin"
 HKCU,Software\Wine\DllOverrides,"msvcp100",0x2,"native,builtin"

